core/tests/puppeteer-acceptance-tests/utilities/user/logged-out-user.ts

```diff
@@ -1382,6 +1431,111 @@ export class LoggedOutUser extends BaseUser {
     }
   }
 
+  /**
+   * Clicks on the Privacy Policy link in the Terms of Service page and
+   * checks if it opens the correct URL.
+   */
+  async clickPrivacyPolicyLinkInTosPage(): Promise<void> {
+    await this.page.waitForSelector(privacyPolicyLinkInTosPage, {
+      visible: true,
+    });
+    await this.clickButtonToNavigateToNewPage(
+      privacyPolicyLinkInTosPage,
+      'Privacy Policy link in the Terms of Service page',
+      privacyPolicyUrl,
+      'Privacy Policy'
+    );
+  }
+
+  /**
+   * Clicks on the License link in the Terms of Service page and checks
+   * if it opens the correct URL.
+   */
+  async clickLicenseLinkInTosPage(): Promise<void> {
+    await this.page.waitForSelector(ccLicenseLinkInTosPage, {visible: true});
+    await this.clickButtonToNavigateToNewPage(
+      ccLicenseLinkInTosPage,
+      'License link in the Terms of Service page',
+      ccLicenseUrl,
+      'License'
+    );
+  }
+
+  /**
+   * Clicks on the Google Group Sign Up link in the Terms of Service page and checks
+   * if it opens the correct URL.
+   */
+  async clickGoogleGroupSignUpLinkInTosPage(): Promise<void> {
+    await this.page.waitForSelector(googleGroupSignUpLinkInTosPage, {
+      visible: true,
+    });
+    await this.clickButtonToNavigateToNewPage(
+      googleGroupSignUpLinkInTosPage,
+      'Google Group Sign Up link in the Terms of Service page',
+      OppiaAnnounceGoogleGroupUrl,
+      'Google Group Sign Up'
+    );
+  }
+
+  /**
+   * Clicks the "DONATE TODAY" button on the Contact Us page and checks that
+   * it navigates to the correct URL.
+   */
+  async clickDonateTodayButtonInContactUsPage(): Promise<void> {
+    await this.clickLinkAnchorInSameTab('DONATE TODAY', donateUrl);
+  }
+
+  /**
+   * Clicks the "BECOME A PARTNER" button on the Contact Us page and checks that
+   * it navigates to the correct URL.
+   */
+  async clickBecomeAPartnerButtonInContactUsPage(): Promise<void> {
+    await this.clickLinkAnchorInSameTab('BECOME A PARTNER', partnershipsUrl);
+  }
+
+  /**
+   * Clicks the "VOLUNTEER WITH US" button on the Contact Us page and checks that
+   * it navigates to the correct URL.
+   */
+  async clickVolunteerButtonInContactUsPage(): Promise<void> {
+    await this.clickLinkAnchorInSameTab('BECOME A VOLUNTEER', volunteerUrl);
+  }
+
+  /**
+   * Clicks the first admin email link in the Contact Us page and checks that it navigates
```

Comment:
I don't see you click the email in this function. Maybe rename it to "checkAdminEmailLinkInContactUsPage" or "verifyAdminEmailLinkInContactUsPage" is better.

Reply:
Ditto for the one below.

Reply:
Updated.